modules/passkey/contracts/WebAuthnSignerFactory.sol

```diff
@@ -26,7 +26,7 @@ contract WebAuthnSignerFactory is ICustomECDSASignerFactory {
     function createSigner(uint256 x, uint256 y, address verifier) external returns (address signer) {
         signer = getSigner(x, y, verifier);
 
-        if (_hasNoCode(signer) && _validVerifier(verifier)) {
+        if (_hasNoCode(signer)) {
```

Comment:
This removal was because if it didn't have any code, the `signer` wouldn't have been created. Right?

Reply:
Updating the PR description.